examples/gatsby-site/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-hello-world",
```

Comment:
`name` should be consistent with the folder name `gatsby-site`. This allows us to execute `yarn workspace gatsby-site develop` from the command line.